### PR TITLE
Add a section <More binding points> in WebGL 2 spec explicitly, considering that some APIs can support more binding points in WebGL 2

### PR DESCRIPTION
**Pull request by Richard-Yunchao**
PTAL when you have time. Thanks!

### CODE REVIEW

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -830,6 +830,58 @@ <h2 class="no-toc">Table of contents</h2>
...
+          <table>
+              <tr><th>target</th></tr>
+              <tr><td>ARRAY_BUFFER</td></tr>
+              <tr><td>ELEMENT_ARRAY_BUFFER</td></tr>
+              <tr><td>PIXEL_PACK_BUFFER</td></tr>
+              <tr><td>PIXEL_UNPACK_BUFFER</td></tr>
+              <tr><td>TRANSFORM_FEEDBACK_BUFFER</td></tr>
+              <tr><td>UNIFORM_BUFFER</td></tr>
+          </table>
+          <p>If <em>target</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: This must refer to the section farther down in the spec "Buffer Object Binding", which defines restrictions, similarly to the bindBuffer description in the WebGL 1.0 spec.

File: specs/latest/2.0/index.html [hunk 2]
```diff
@@ -830,6 +830,58 @@ <h2 class="no-toc">Table of contents</h2>
...
+      <dd>
+          Binds the given WebGLFramebuffer object to the given binding point(target). <em>target</em> is given in the following table:
+          <table>
+              <tr><th>target</th></tr>
+              <tr><td>FRAMEBUFFER</td></tr>
+              <tr><td>READ_FRAMEBUFFER</td></tr>
+              <tr><td>DRAW_FRAMEBUFFER</td></tr>
+          </table>
+          <p>If <em>target</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
+      </dd>
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: This description should include the same constraints about the default framebuffers as described in the WebGL 1.0 version of the spec.

**Comment by kenrussell** [comment]
Thanks for this contribution but needs a little more work.

**Comment by kenrussell** [comment]
@jdashg would you like to take over the review of this pull request?

**Comment by jdashg** [comment]
Sure.

**Comment by Richard-Yunchao** [comment]
Addressed kbr@'s feedback. @kenrussell and/or @jdashg , PTAL when you have time. Thanks a lot!

**Comment by kenrussell** [comment]
I defer to @jdashg 's review.

**Comment by Richard-Yunchao** [comment]
No problem.

**Comment by Richard-Yunchao** [comment]
Ping... @jdashg , could you have a look? Thanks a lot!

**Comment by jdashg** [comment]
That's fine. There's some copy which could be better, but we can fix it later.

LGTM.

**Comment by kenrussell** [comment]
@Richard-Yunchao : could you please rebase this pull request? It can no longer be merged automatically. Thanks.

**Comment by Richard-Yunchao** [comment]
@kenrussell , sorry to be late. The pull request is rebased to the lastest now. PTAL.

**Comment by kenrussell** [comment]
Thanks. Merging based on @jdashg 's review.

**Pull request merged**